packages/serve/src/lib/catalog-router/catalogRouters.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  TYPES as CORE_TYPES,
+  VulcanArtifactBuilder,
+  VulcanExtensionId,
+  VulcanInternalExtension,
+  ICoreOptions,
+  APISchema,
+} from '@vulcan-sql/core';
+import { Next } from 'koa';
+import * as Router from 'koa-router';
+import { CatalogRouter } from '@vulcan-sql/serve/models';
+import { inject } from 'inversify';
+import { getDocUrlPrefix } from '../document-router/utils';
+@VulcanInternalExtension('catalog')
+@VulcanExtensionId('catalog')
```

Comment:
Seems we only have `CatalogRouter` now, and our `CatalogRouter` won't be created by the factory and injected into other classes, so the `VulcanExtensionId`  won't be work, because `VulcanExtensionId`  only used when we need to create the instance by inversify factory or you get the `extensionId` by hand to do somethings, so here I may suggest you remove it to prevent other members confusion :)

Reply:
Btw, I discovered our `RedocDocumentRounter` seems to set the `VulcanExtensionId` and it actually created by Inversify `AutoNamedFactory`, so the `DocumentRouter` actually should add the `{ enforceId: true }`, but seems it did not add before, so could you help me to add the `{ enforceId: true }` ? Thanks :D